sdk/python/feast/feature_store.py

```diff
@@ -1321,9 +1354,14 @@ def write_to_online_store(
         ingests data directly into the Online store
         """
         # TODO: restrict this to work with online StreamFeatureViews and validate the FeatureView type
-        feature_view = self.get_feature_view(
-            feature_view_name, allow_registry_cache=allow_registry_cache
-        )
+        try:
+            feature_view = self.get_stream_feature_view(
+                feature_view_name, allow_registry_cache=allow_registry_cache
+            )
+        except FeatureViewNotFoundException:
+            feature_view = self.get_feature_view(
+                feature_view_name, allow_registry_cache=allow_registry_cache
+            )
```

Comment:
IMO we should try to build on top of #2733 to migrate these kinds of stray feature views to the right location.

Reply:
Hm I see, I'll add a todo for this.